tests/auxtel/test_cam_take_image.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import asyncio
+
+import yaml
+
+from lsst.ts import salobj
+
+from lsst.ts.standardscripts.auxtel.atcam_take_image import ATCamTakeImage
+
+import SALPY_ATCamera
+import SALPY_Script
+
+index_gen = salobj.index_generator()
+
+
+class Harness:
+    def __init__(self):
+        self.index = next(index_gen)
+        self.test_index = next(index_gen)
+        salobj.test_utils.set_random_lsst_dds_domain()
+        self.script = ATCamTakeImage(index=self.index)
+        self.atcam = salobj.Controller(SALPY_ATCamera)
```

Comment:
This is a great start. Some suggestions for a bit more:
- Add controllers for the linear stage, filterwheel, etc., and use callbacks to accumulate information about calls (e.g. a list of requested positions, one per command)
- Accumulate some information in cmd_take_images_callback, such as the requested exposure time, then check it.

Also consider wiring the cmd_take_images_callback to self.atcam in the constructor.